pkg/devconsole/webhooks/webhooks.go

```diff
@@ -0,0 +1,136 @@
+package webhooks
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"slices"
+
+	"github.com/openshift/console/pkg/auth"
+	"github.com/openshift/console/pkg/devconsole/common"
+)
+
+var headerDenyList = []string{
+	"Cookie",
+	"X-CSRF-Token",
+}
```

Comment:
I'm a little hesitant to define this in two places since it's also in main.go. If we ever need to add another header to the blacklist, it would be easy to miss this spot.

https://github.com/openshift/console/blob/main/cmd/bridge/main.go#L355

Can this be passed to in as part of the `Server` config so we can define the list in one place in main.go?